app/src/main/java/com/codesoom/assignment/service/TaskService.java

```diff
@@ -0,0 +1,77 @@
+package com.codesoom.assignment.service;
+
+import com.codesoom.assignment.exception.TaskNotFoundException;
+import com.codesoom.assignment.model.Task;
+import org.springframework.stereotype.Service;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+@Service
+public class TaskService {
+
+    private Map<Long, Task> tasks = new LinkedHashMap<Long, Task>();
```

Comment:
레퍼런스가 날라가는 일을 방지해두면 좀 더 튼튼한 코드가 되겠죠.
```suggestion
    private final Map<Long, Task> tasks = new LinkedHashMap<Long, Task>();
```